template/build_command.template.dart

```diff
@@ -29,17 +47,19 @@ class BuildCommand extends Command {
     // You can change the hashType to sha384 or sha512 if you want
 
     hashScripts(hashType: sha256);
+    enforceCSP(shouldEnforce: env.shouldEnforceCSP);
 
     // You can insert your own logic here after moving the Flutter app to the server directory (packages/server/www)
     // and before building the Docker image
 
-    createDockerImage();
+    createDockerImage(env.name);
 
-    // TODO: Remove this warning in production
-    print(
-      yellow(
-        'Warning: Update the CSP Rules in the template/middlewares.template.dart file to make the app production ready',
-      ),
-    );
+    if (env.name != 'prod') {
```

Comment:
Do we want to show this always on prod or if
`env.name != 'prod' && !env.shouldEnforceCSP` or something along the lines?

Reply:
Fixed it with a TODO Comment to better state out what we actually want from the user